pkg/cmd/factory/default.go

```diff
@@ -293,6 +293,12 @@ func ioStreams(f *cmdutil.Factory) *iostreams.IOStreams {
 		io.SetPager(pager.Value)
 	}
 
+	if _, ghColorLabels := os.LookupEnv("GH_COLOR_LABELS"); ghColorLabels {
+		io.SetColorLabels(true) // TODO: should this be a truthy value?
```

Comment:
I think _probably_ just for conventions sake of being able to disable something that is otherwise configured on? Like if you have it configured in config and then want to disable it.